src/mumble/UserModel.cpp

```diff
@@ -540,8 +541,16 @@ QVariant UserModel::data(const QModelIndex &idx, int role) const {
 				if (!c->qbaDescHash.isEmpty())
 					l << (item->bCommentSeen ? qiCommentSeen : qiComment);
 
-				if (c->bFiltered)
-					l << (qiFilter);
+				switch (c->m_filterMode) {
+					case Channel::FILTER_MODE_HIDE:
+						l << (qiFilter);
+						break;
+					case Channel::FILTER_MODE_PIN:
+						l << (qiPin);
+						break;
+					default:
+						break;
```

Comment:
```suggestion
					case Channel::FILTER_MODE_NORMAL:
						break;
```
This way, this code will generate a warning if the filter enum is ever extended and that new case is not yet handled here. With a `default` clause, the compiler will never generate such a warning.